scrapers/ak/bills.py

```diff
@@ -120,21 +140,20 @@ def scrape_bill(
                     if sponsor:
                         bill.add_sponsorship(
                             sponsor,
+                            chamber=sponsor_chamber,
                             entity_type="person",
                             classification="cosponsor",
                             primary=False,
                         )
 
-            else:
+            if sponsors_cnt == 0:
                 # Committee sponsorship
                 spons_str = spons_str.strip()
-
                 if re.match(r" BY REQUEST OF THE GOVERNOR$", spons_str):
```

Comment:
Looks like about half of the organizational sponsorships still include the `by request...` language in the `name` property. Can you please check this and adjust logic to try to strip it out in all cases?

```
jesse@greenbookwork:~/repo/openstates/braykuka/openstates-scrapers/scrapers/_data/ak$ rg -ic "by request" | wc -l
122
```

```
 SELECT bill_identifier, "name", "primary", organization_id  FROM bill_sponsorships WHERE entity_type = 'organization' AND lower(name) LIKE '%request%' ORDER BY bill_identifier, name;
┌─────────────────┬────────────────────────────────────────────────┬─────────┬───────────────────────────────────────────────────────────────────┐
│ bill_identifier │                      name                      │ primary │                          organization_id                          │
├─────────────────┼────────────────────────────────────────────────┼─────────┼───────────────────────────────────────────────────────────────────┤
│ HB105           │ HOUSE RULES BY REQUEST OF THE GOVERNOR         │ true    │ "~{\"name\": \"HOUSE RULES BY REQUEST OF THE GOVERNOR\"}"         │
│ HB106           │ HOUSE RULES BY REQUEST OF THE GOVERNOR         │ true    │ "~{\"name\": \"HOUSE RULES BY REQUEST OF THE GOVERNOR\"}"         │
│ HB119           │ HOUSE RULES BY REQUEST                         │ true    │ "~{\"name\": \"HOUSE RULES BY REQUEST\"}"                         │
│ HB140           │ HOUSE RULES BY REQUEST                         │ true    │ "~{\"name\": \"HOUSE RULES BY REQUEST\"}"                         │
│ HB146           │ HOUSE RULES BY REQUEST OF THE GOVERNOR         │ true    │ "~{\"name\": \"HOUSE RULES BY REQUEST OF THE GOVERNOR\"}"         │
```

so:

- `HOUSE RULES BY REQUEST OF THE GOVERNOR` should just be `HOUSE RULES`
- `HOUSE RULES BY REQUEST` should just be `HOUSE RULES`
- `SENATE RULES BY REQUEST OF LEGISLATIVE COUNCIL` should just be `SENATE RULES`